pest_meta/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+#[cfg(test)]
+#[macro_use]
+extern crate pest;
+
+#[cfg(not(test))]
+extern crate pest;
+
+#[macro_use]
+extern crate pest_derive;
+
+extern crate maplit;
+
+pub mod parser;
+pub mod ast;
+pub mod validator;
+
+use std::fmt::Display;
+
+pub fn unwrap_or_report<T, E>(res: Result<T, E>) -> T where E: IntoIterator, E::Item: Display {
+    res.unwrap_or_else(|e| panic!("grammar error\n\n".to_owned() + &e.into_iter()
+              .map(|error| { format!("{}", error) })
```

Comment:
Not sure about this formatting. Maybe a newer version of rustfmt needed?

Reply:
I will use `rustfmt` on this file. Would you recommend using `cargo fmt` to format all 4 crates? That would create a lot of changes, and in some places arguably wrong formatting. I feel like rustfmt is not really ready for prime time yet.

Reply:
Yeah, `rustfmt` is not ready for prime-time. :/ I'm still kind of torn between whether or not to use it, but for now, I guess it's fine. I expect formatting to get better in time. Anyway, I definitely don't mind the changes if they're concentrated in a last commit.

Reply:
We could do a `cargo fmt` as part of a dedicated PR which only deals with formatting. Merging (non-fast-forward) can be quite difficult after such changes since there would be conflicts on so many lines.